### PR TITLE
DCD-53: Update Jira DC quickstart templates

### PR DESCRIPTION
**Pull request by tarka**

### CODE REVIEW

File: templates/quickstart-jira-dc-with-vpc.template.yaml [hunk 1]
```diff
@@ -0,0 +1,728 @@
+---
+# Bring up diffs between this file and the BitbucketDataCenter.template.yaml, ConfluenceDataCenter.template.yaml, and CrowdDataCenter.template.yaml
+# As a rule, we should work to minimize our diffs between these files as best we can, so that future changes are easy to make across all supported CloudFormation templates.
+# Using YAML as our file format will allow us to put a block comment at the top of the file saying exactly this.
+AWSTemplateFormatVersion: 2010-09-09
+Description: Atlassian Jira Data Center
```
Comment (nanux) [inline templates/quickstart-jira-dc-with-vpc.template.yaml]: This should reflect the template structure. In this case, the Jira is spun up with VPC as well.
```suggestion
Description: Atlassian Jira Data Center with VPC
```

File: templates/quickstart-jira-dc-with-vpc.template.yaml [hunk 2]
```diff
@@ -0,0 +1,728 @@
...
+        default: Tomcat Maximum Threads
+      TomcatMinSpareThreads:
+        default: Tomcat Minimum Spare Threads
+      TomcatProtocol:
+        default: Tomcat Protocol
+      TomcatRedirectPort:
+        default: Tomcat Redirect Port
+      TomcatScheme:
+        default: Tomcat protocol Scheme
+
```
Comment (nanux) [inline templates/quickstart-jira-dc-with-vpc.template.yaml:189]: There are several empty lines through this file, are they intentional?

File: templates/quickstart-jira-dc-with-vpc.template.yaml [hunk 3]
```diff
@@ -0,0 +1,728 @@
...
+        TomcatScheme: !Ref 'TomcatScheme'
+
+Outputs:
+  ServiceURL:
+    Description: The URL to access this Atlassian service
+    Value: !GetAtt 'JiraDCStack.Outputs.ServiceURL'
+  LoadBalancerURL:
+    Description: The Load Balancer URL
+    Value: !GetAtt 'JiraDCStack.Outputs.LoadBalancerURL'
+  SGname:
```
Comment (nanux) [inline templates/quickstart-jira-dc-with-vpc.template.yaml:719]: When we are creating the VPC stack as part of the template I found it very valuable for testing and further investigation to add `BastionPubIp` from the Bastion stack. This is now enabled with this change being merged -https://github.com/aws-quickstart/quickstart-atlassian-services/blob/master/templates/quickstart-vpc-for-atlassian-services.yaml#L226 (Outputs doesn't support referencing value from the second level).
Something like
```suggestion
  BastionIP:
    Description: Bastion node IP (use as a jumpbox to connect to the nodes)
    Value: !GetAtt 'VPCStack.Outputs. BastionPubIp'
```

**Review by nanux — APPROVED** [review]
LGTM. Most of it was already reviewed in the Bitbucket repo and I've listed a couple of suggestions that are worth considering before merging.

File: templates/quickstart-jira-dc-with-vpc.template.yaml (hunk 2 above)
Comment (tarka) [reply templates/quickstart-jira-dc-with-vpc.template.yaml:189]: They are. I prefer to group the related functionality together in config/template files, much as you would with code.  It also makes jumping around between sections in the editor easier.

**Pull request merged**